### PR TITLE
Use case

### PR DESCRIPTION
**Pull request by MrGKanev**

### CODE REVIEW

**Comment by MrGKanev** [comment]
13:51 push, изпит 14:00 ...

File: resources/views/themes/bimbala/partials/footer.blade.php [hunk 1]
```diff
@@ -38,10 +38,11 @@
                         Product
                     </h2>
                     <ul class="p-0 m-0 text-base leading-tight list-none">
-                    {{--    <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
-                            <a href="#_" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Drag'n Drop Builder</a>
-                        </li>
                         <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
+                            <a href="{{ route('use-case') }}"
+                                class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Use Case</a>
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/partials/footer.blade.php:43]: Let's keep the wording consistent
```suggestion
                                class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Use Cases</a>
```

File: resources/views/themes/bimbala/partials/footer.blade.php [hunk 2]
```diff
@@ -20,14 +20,14 @@
                         <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
                             <a href="{{ route('try-it') }}" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Try it</a>
                         </li>
-                        {{--  <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
-                            <a href="#_" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Changelog</a>
+                        {{-- <li class="box-border py-2 text-sm font-normal text-left text-gray-500 md:text-base md:mb-1">
+                            <a href="{{ route('use-case') }}" class="text-base leading-tight no-underline bg-transparent cursor-pointer hover:text-gray-400">Use Case</a>
```
Comment (l-alexandrov) [inline resources/views/themes/bimbala/partials/footer.blade.php:24]: Is this change really necessary?

**Review by l-alexandrov — CHANGES_REQUESTED** [review]

File: resources/views/themes/bimbala/partials/footer.blade.php (hunk 2 above)
Comment (MrGKanev) [reply resources/views/themes/bimbala/partials/footer.blade.php]: My idea was to place it in the correct "category" or at least to use the categories which we have made for now. I changed the "old" one because it will be used for something else. I'm open for discussion if necessary.

Comment (l-alexandrov) [reply resources/views/themes/bimbala/partials/footer.blade.php]: I see.

**Review by l-alexandrov — APPROVED** [review]
